services/src/main/java/org/keycloak/credential/WebAuthnPasswordlessCredentialProvider.java

```diff
@@ -57,4 +63,27 @@ public CredentialTypeMetadata getCredentialTypeMetadata(CredentialTypeMetadataCo
     protected WebAuthnPolicy getWebAuthnPolicy() {
         return getKeycloakSession().getContext().getRealm().getWebAuthnPolicyPasswordless();
     }
+
+    @Override
+    public CredentialMetadata getCredentialMetadata(WebAuthnCredentialModel credentialModel, CredentialTypeMetadata credentialTypeMetadata) {
+
+        CredentialMetadata credentialMetadata = new CredentialMetadata();
+
+        try {
+            WebAuthnCredentialData credentialData = JsonSerialization.readValue(credentialModel.getCredentialData(), WebAuthnCredentialData.class);
+            Set<String> transports = credentialData.getTransports();
+            if (transports != null && !transports.isEmpty()) {
+                String joinedTransports = String.join(", ", transports);
+                credentialMetadata.setInfoMessage("credentialTransports", joinedTransports);
```

Comment:
Just a cherry-pick, but maybe you can use a static private property in the model like it is used in the recovery-codes (see [here](https://github.com/keycloak/keycloak/blob/26.3.2/services/src/main/java/org/keycloak/credential/RecoveryAuthnCodesCredentialProvider.java#L82))? And I would use something more generic like `webauthn-info` just in case we include something more later...